script/Ciel/Day 6/06_01_CIEL06_1_1.txt

```diff
@@ -131,6 +131,7 @@ I sit down on the sofa, doing my best to endure the pain running through my enti
 [sha:84255c3b15e88d7018db00d48a81719184862361]{
 -- Page 46, Offset 13269. Mods: @k, @e.
 -- 「卑怯でしょう。そんな顔をされたら気がゆる―――
+// "Of course" is not the correct response to "why." Requesting a JP speaker to suggest alternative to this line or the previous
```

Comment:
I guess you could change it to "isn't it obvious" or something? It drops the 'unfair' but it's in the spirit imo. Or rewrite more to something like
> [Huh?] How is it unfair?
> How isn't it unfair! If you...